python/dgl/distributed/graph_services.py

```diff
@@ -80,7 +80,9 @@ def _sample_etype_neighbors(local_g, partition_book, seed_nodes, etype_field,
     """
     local_ids = partition_book.nid2localnid(seed_nodes, partition_book.partid)
     local_ids = F.astype(local_ids, local_g.idtype)
-
+    sort_etypes = os.environ.get('DGL_SORT_ETYPES', '')
```

Comment:
This is really bad practice. Please do not rely on environment variable besides the very entry point. This is very confusing and difficult for others to understand when and where this flag is set. Please pass it down by function argument.